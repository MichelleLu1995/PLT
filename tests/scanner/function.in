return
void
file
print
main
def
in
