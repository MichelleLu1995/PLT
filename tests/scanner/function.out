RETURN
VOID
FILE
PRINT
MAIN
DEF
IN
